tests/datasetslice_test.py

```diff
@@ -1,129 +0,0 @@
-        d1['y'] = sc.Variable(dims=['y'], values=np.arange(M).astype(np.float64))
-        arr1 = np.arange(N * M).reshape(N, M).astype(np.float64) + 1
-        d1['a'] = sc.Variable(dims=['x', 'y'], values=arr1)
-        d1 = d1['x', 1:2]
-        assert d1['a'].data.values.tolist() == [[5.0, 6.0, 7.0, 8.0]]
-
-    def test_set_dataarrayview_slice_items(self):
-        d = self._d.copy()
-        d['a']['x', 0:2] += d['b']['x', 0:2]
-        assert d['a'].data.values.tolist() == [0, 2, 2, 3, 4]
-        d['a']['x', 4] += \
-            d['b']['x', 1]
-        assert d['a'].data.values.tolist() == [0, 2, 2, 3, 5]
-
-    def test_slice_and_dimensions_items_dataarray(self):
-        var = sc.Variable(dims=['x', 'y'], values=np.arange(50).reshape(5, 10))
-        da = sc.DataArray(var)
-        assert np.allclose(da['x', 0].values, da['x', 0:1].values)
-        assert np.allclose(da['x', 4].values, da['x', -1].values)
-        assert np.allclose(da['y', 1].values, da['y', -9].values)
-        assert ('y' in da['x', 0].dims)
-        assert ('x' not in da['x', 0].dims)
-        assert ('y' in da['x', 0:1].dims)
-        assert ('x' in da['x', 0:1].dims)
-
-    def test_slice_and_dimensions_items_dataset(self):
-        da = sc.DataArray(
-            sc.Variable(dims=['x', 'y'], values=np.arange(50).reshape(5, 10)))
-        ds = sc.Dataset(data={'a': da})
-        assert (np.allclose(ds['x', 0]['a'].values,
-                            ds['x', 0:1]['a'].values[0],
-                            atol=1e-9))
-        assert (np.allclose(ds['x', 4]['a'].values, ds['x', -1]['a'].values))
-        assert (np.allclose(ds['y', 1]['a'].values, ds['y', -9]['a'].values))
-        assert ('y' in da['x', 0].dims)
-        assert ('x' not in da['x', 0].dims)
-        assert ('y' in da['x', 0:1].dims)
-        assert ('x' in da['x', 0:1].dims)
-
-    def test_slice_dataset_with_data_only(self):
-        d = sc.Dataset()
-        d['data'] = sc.Variable(dims=['y'], values=np.arange(10))
-        sliced = d['y', :]
-        assert sc.identical(d, sliced)
-        sliced = d['y', 2:6]
-        assert sc.identical(sc.Variable(dims=['y'], values=np.arange(2, 6)),
-                            sliced['data'].data)
-
-    def test_slice_dataset_with_coords_only(self):
-        d = sc.Dataset(
-            coords={'y-coord': sc.Variable(dims=['y'], values=np.arange(10))})
-        sliced = d['y', :]
-        assert sc.identical(d, sliced)
-        sliced = d['y', 2:6]
-        assert sc.identical(sc.Variable(dims=['y'], values=np.arange(2, 6)),
-                            sliced.coords['y-coord'])
-
-    def test_slice_with_step_1(self):
-        var = sc.Variable(dims=['x'], values=np.arange(1, 4, dtype=np.int64))
-        expect = sc.Dataset(data={'a': var, 'b': var}, coords={'x': var})
-        assert sc.identical(self._d['x', 1:4:1], expect)
-        assert sc.identical(self._d['a']['x', 1:4:1], expect['a'])
```

Comment:
I think I agree that all these can be removed: This should have been tested on C++ side, and is not related to the Python bindings.